Add Delphinet support

sean-magin opened the pull request:
- Added delphinet support
- Updated types in all components

claudebarde reviewed (commented):
In ConnectWallet.tsx, shouldn't the RPC URL be also updated to Delphinet? I think Taquito's node has been updated already.

sean-magin commented:
> In ConnectWallet.tsx, shouldn't the RPC URL be also updated to Delphinet? I think Taquito's node has been updated already.

![Screen Shot 2020-11-22 at 11 19 02 AM](https://user-images.githubusercontent.com/14209806/99888849-a510a900-2cb4-11eb-9f15-4f05bcb75ed5.png)

Yeah, I've already updated that to use Delphinet. Just to double check is that the rpcUrl you're mentioning?

claudebarde commented:
This one in commit d20afdf:
![image](https://user-images.githubusercontent.com/25201109/99889424-de153300-2c4c-11eb-81eb-68162181aba6.png)

sean-magin commented:
> This one in commit [d20afdf](https://github.com/ecadlabs/taquito-react-template/commit/d20afdfe6a3a86ae998e3e7c0e106da18a3dad14):
> ![image](https://user-images.githubusercontent.com/25201109/99889424-de153300-2c4c-11eb-81eb-68162181aba6.png)

That's pretty weird, b/c if you click on the `Files Changed` tab it shows that I have in fact updated it to Delphi, yet the latest commit says carthage? Strange one
